src/client/client_impl.rs

```diff
@@ -41,13 +41,21 @@ unsafe impl Sync for Client {}
 
 impl Client {
     /// Opens a JACK client with the given name and options. If the client is successfully opened,
-    /// then `Ok(client)` is returned. If there is a failure, then `Err(Error::ClientError(status))`
-    /// will be returned.
+    /// then `Ok(client)` is returned. If the JACK server returned an error, then
+    /// `Err(Error::ClientError(status))` will be returned. And if the JACK library could not be
+    /// loaded, an `Err(Error::LibraryError(message))` is returned.
     ///
     /// Although the client may be successful in opening, there still may be some errors minor
     /// errors when attempting to opening. To access these, check the returned `ClientStatus`.
     pub fn new(client_name: &str, options: ClientOptions) -> Result<(Self, ClientStatus), Error> {
         let _m = CREATE_OR_DESTROY_CLIENT_MUTEX.lock().unwrap();
+
+        // All of the jack_sys functions below assume the client library is loaded and will panic if
```

Comment:
Can you guard the execution of this around `#[cfg(feature = "dynamic_loading")]` ?

Reply:
Done